packages/appstore/src/components/add-more-wallets/carousel-buttons.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+
+type PrevNextButtonProps = {
+    enabled: boolean;
+    onClick: () => void;
+};
+
+export const PrevButton: React.FC<PrevNextButtonProps> = props => {
+    const { enabled, onClick } = props;
+
+    return (
+        <button className='carousel__btn carousel__btn-prev' onClick={onClick} disabled={!enabled}>
+            <Icon icon='IcChevronLeftBold' className='carousel-btn__icon' />
+        </button>
+    );
+};
+
+export const NextButton: React.FC<PrevNextButtonProps> = props => {
```

Comment:
same as above. Probably could make prev and next button into one small component all together by just passing a prop into it.